Makefile

```diff
@@ -149,6 +149,34 @@ examples/output: src/mixs/schema/mixs.yaml
 		--output-directory $@ \
 		--schema $< > $@/README.md
 
+.PHONY: standardize-schema
+
+standardize-schema:
+	$(RUN) python src/scripts/describe_enums_by_slots_using.py \
```

Comment:
Does this script account for the fact that if enum descriptions are already populated in the schema, then don't autopopulate with a boilerplate description?

Reply:
Resolved.